spec/spec_helper.rb

```diff
@@ -1,3 +1,23 @@
 require 'coveralls'
+require 'simplecov'
+
+SimpleCov.formatters = [
+  SimpleCov::Formatter::HTMLFormatter,
+  Coveralls::SimpleCov::Formatter
+]
 Coveralls.wear!
+#SimpleCov.start
 require 'capybara/rspec'
+
+
+RSpec.configure do |config|
+
+  config.expect_with :rspec do |expectations|
+     expectations.include_chain_clauses_in_custom_matcher_descriptions = true
+   end
```

Comment:
Lint/BlockAlignment: `end` at 17, 3 is not aligned with `config.expect_with :rspec do |expectations|` at 15, 2